src/azure-cli-core/azure/cli/core/azclierror.py

```diff
@@ -249,4 +251,10 @@ class CLIInternalError(ClientError):
     """ AzureCLI internal error """
     pass
 
+
+# Client error for az next
+class RecommendationError(ClientError):
```

Comment:
If this error is dedicated to `az next`, maybe consider putting it in `next` extension?

Reply:
Yes,  it is only used in `log_latest_error_info` at present. When `log_latest_error_info` is migrated to extension, it will also be migrated